### DISCUSSION TITLE
Notification with line break

### DISCUSSION THREAD

**Discussion by russocorp** [mantinedev/mantine#3429] [Q&A] [answered]
Hi, i´m using this to show notification to users:

showNotification({
        autoClose: 3000,
        message: text,
        color: 'yellow',
        disallowClose: true,
    });

How can i pass value to text variable to break line?
I tried \n but not work.

**Comment by Sajarin-M** [comment, marked as answer]
You can pass a react node as `message` prop like:
```js
showNotification({
  autoClose: 3000,
  message: (
    <>
      <Text>line 1</Text>
      <Text>line 2</Text>
    </>
  ),
  color: 'yellow',
  disallowClose: true,
});
```
